<?xml version="1.0"?>
<launch>
<!-- PipetteTool -->
    <node name="pipette_200ul" pkg="ros_robotic_tools" type="pipette_tool_action_server" output="screen">
        <param name="baudrate" type="int" value="9600"/>
        <param name="device" type="str" value="/dev/pipette_tool"/>
        <param name="action_server_topic" type="str" value="/udes_pipette"/>
    </node>
    </launch>
